dataactcore/utils/jobQueue.py

```diff
@@ -37,6 +71,8 @@ def __init__(self, job_queue_url="localhost"):
         self.jobQueue.config_from_object('celeryconfig')
 
         self.enqueue = self.jobQueue.task(name='jobQueue.enqueue')(enqueue)
+        self.generate_f_file = self.jobQueue.task(
```

Comment:
I don't understand the job queue very well and wasn't the D file reviewer, so forgive the question...what does this line of code do? Is the `generate_f_file` at the end some kind of callback?